src/templater.sh

```diff
@@ -48,6 +48,7 @@ function templater() {
 
   # cat "${templateDst}"
 
+  # shellcheck disable=SC2016
   if grep '${' "${templateDst}" && [ "${ALLOW_REMAINDERS}" = "0" ]; then
```

Comment:
But there is this intermittent failure. A restart sometimes fixes it:

![screen shot 2016-12-15 at 19 54 51](https://cloud.githubusercontent.com/assets/26752/21237708/61280e4e-c300-11e6-96bc-f4aac4d3f6ea.png)